src/MOL-SLLOD/fix_nvt_sllod_mol.cpp

```diff
@@ -19,31 +19,49 @@
 #include "fix_nvt_sllod_mol.h"
```

Comment:
Also probably want to update the contributing author to add Stephen and Shern here